Makefile

```diff
@@ -72,6 +72,14 @@ test-cross:
 	$(MAKE) -j 3 GOOS=linux tmp_moby_bin.tar tmp_rtf_bin.tar tmp_mt_bin.tar tmp_linuxkit_bin.tar
 	$(MAKE) clean
 
+ifeq ($(GOARCH)-$(GOOS),amd64-linux)
+LOCAL_BUILDMODE?=pie
```

Comment:
I guess cross building static pie for arm64-linux doesnt work as there is no cross linker?

Reply:
The failure was in https://circleci.com/gh/linuxkit/linuxkit/25 and I think your diagnosis is correct.